tests/unit/test_pipelines.py

```diff
@@ -61,13 +61,10 @@ def expected_response():
 
 
 def test_prepare_payload(
-    tmpdir, json_spider_record, spider, expected_response,
+    tmpdir, json_spider_record, spider, expected_response, set_up_clean_up_env_vars,
 ):
     """Test that the generated payload is ok."""
     _, json_record = json_spider_record
-    os.environ['SCRAPY_JOB'] = 'scrapy_job'
-    os.environ['SCRAPY_FEED_URI'] = 'scrapy_feed_uri'
-    os.environ['SCRAPY_LOG_FILE'] = 'scrapy_log_file'
```

Comment:
You should leave this here. That way, you know that this test depends on those env variables having those values.